src/test/scala/ch/epfl/bluebrain/nexus/tests/BaseSpec.scala

```diff
@@ -45,7 +45,7 @@ class BaseSpec
   private[tests] val credGroup                     = HttpCredentials.createOAuth2BearerToken(config.iam.groupToken)
   private[tests] val credUser                      = HttpCredentials.createOAuth2BearerToken(config.iam.userToken)
   private[tests] val headersGroup: Seq[HttpHeader] = Seq(Authorization(credGroup))
-  private[tests] val headersUserAcceptJson: Seq[HttpHeader] =
+  private[tests] val headerUser: Seq[HttpHeader] =
```

Comment:
another variable is already called `headersUser`, we should rename it to something which is more easily distinguishable than `headerUser` and `headersUser`